assignments/prototypes.js

```diff
@@ -7,8 +7,42 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
+  const GameObject = function(attributes){
+    this.createdAt = 'June52019',
+    this.name = attributes.name,
+    this.dimensions = attributes.dimensions
+    this.destroy = function() {
+      return `${this.name} was removed from the game.`}
+  }
+GameObject.prototype.destroy = function () {
+}
```

Comment:
We would want to put that function logic in here. We can still use `this` and everything!